release-please-config.json

```diff
@@ -4,7 +4,8 @@
     "packages/react": {},
     "packages/solid": {},
     "packages/vue": {},
-    "packages/svelte": {}
+    "packages/svelte": {},
+    "packages/astro": {}
```

Comment:
should this also be added to the `build:packages` step, or is that unneeded?

Reply:
I thought the same at first, but no. The package doesn't have a build step, because Astro supports libraries with un-transpiled files.